plugins/generator-1.18.2/forge-1.18.2/procedures/entity_islooking_at_entity.java.ftl

```diff
@@ -0,0 +1,16 @@
+(new Object() {
+	public boolean isLooking(Entity _source, Entity _target, double area, boolean ignoreBlocks) {
+		Vec3 vec3 = _source.getViewVector(1.0F).normalize();
+		Vec3 vec31 = new Vec3(_target.getX() - _source.getX(), (_target.getEyeY() - _target.getEyeHeight() * 0.5) - _source.getEyeY(),
+				_target.getZ() - _source.getZ());
+		double d0 = vec31.length();
+		vec31 = vec31.normalize();
+		double d1 = vec3.dot(vec31);

```

Comment:
You use `vec3` only at this line, so maybe you can simply inline everything.

Reply:
I am guessing you mean inline from line 3 to 8? @Goldorion

Reply:
Actually I am not sure if that's possible, because you need the lenght of unmodified vec31 for d0, right? @Goldorion

Reply:
What I meant is something like this
`_source.getViewVector(1.0F).normalize().dot(vec31)`